README.md

```diff
@@ -312,44 +309,58 @@ metadata:
 spec:
   description: string # optional up to 1050 characters
   thresholdMetric: # either thresholdMetric or ratioMetric must be provided
-    metricSource:
-      metricSourceRef: string # optional, this field can be used to refer to DataSource object
-      type: string # optional, this field describes predefined metric source type e.g. Prometheus, Datadog, etc.
-      spec:
-        # arbitrary chosen fields for every data source type to make it comfortable to use
-        # anything that is valid YAML can be put here.
+    # either dataSourceRef or <<dataSourceName>> must be provided
```

Comment:
These are also backward incompatible changes, would this also require a move to 2.0 for the spec?

Reply:
This is something to discuss at the next community meeting. In my opinion, we should have clearly defined milestones for 2.0.

But on the other hand, I think we can create a dedicated branch e.g. `2.0-alpha` and experiment there